pkg/sysinfo/sysinfo.go

```diff
@@ -50,17 +47,6 @@ func New(quiet bool) *SysInfo {
 		}
 	}
 
-	// Checek if ipv4_forward is disabled.
-	if data, err := ioutil.ReadFile("/proc/sys/net/ipv4/ip_forward"); os.IsNotExist(err) {
```

Comment:
It is checking that your system supports `forwarding` at all. I think probably should be still here.